frameworks/hdfs/tests/test_overlay.py

```diff
@@ -8,23 +8,24 @@
 import sdk_networks as networks
 
 
+@pytest.fixture(scope='module', autouse=True)
+def configure_package(configure_universe):
+    try:
+        install.uninstall(PACKAGE_NAME)
+        utils.gc_frameworks()
+        install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT,
+                        additional_options=networks.ENABLE_VIRTUAL_NETWORKS_OPTIONS)
 
+        yield # let the test session execute
+    finally:
+        install.uninstall(PACKAGE_NAME)
 
-def setup_module(module):
-    install.uninstall(PACKAGE_NAME)
-    utils.gc_frameworks()
-    install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT,
-                    additional_options=networks.ENABLE_VIRTUAL_NETWORKS_OPTIONS)
 
-
-def setup_function(function):
+@pytest.fixture(autouse=True)
```

Comment:
FYI, its entirely a matter of personal preference but instead of having multiple autouse fixtures, you can just have a global at the top of the test file:
```
pytestmark = pytest.mark.usefixtures('configure_package'. 'pre_test_setup')
```
See: https://docs.pytest.org/en/latest/fixture.html#usefixtures